Sources/Error Handling/ErrorUtils.swift

```diff
@@ -471,6 +471,7 @@ private extension ErrorUtils {
 
         Self.logErrorIfNeeded(
             code,
+            message: message,
```

Comment:
Does the message include all the useful information or should we also include the `underlyingError` or any other info?

Reply:
Good question. The underlying error is sometimes printed separately so it could be duplicated.